slamd/discovery/processing/experiment/models/tuned_gaussian_process_regressor.py

```diff
@@ -0,0 +1,63 @@
+from mlxtend.feature_selection import SequentialFeatureSelector
```

Comment:
the package containing the ml models is now called models. I would prefer someting like mlmodel. This way it is clear that these classes are not models in the way we usually use models, i.e. as session objects.